pkg/api/configsync/v1beta1/resource_override.go

```diff
@@ -160,10 +160,12 @@ type ContainerLogLevelOverride struct {
 	// +kubebuilder:validation:Pattern=^(reconciler|git-sync|hydration-controller|oci-sync|helm-sync|gcenode-askpass-sidecar|otel-agent)$
 	ContainerName string `json:"containerName"`
 
-	// logLevel specifies the log level override value for the container.
-	// The default value for git-sync container is 5, while all other containers will default to 0.
-	// Allowed values are from -1 to 10
-	// +kubebuilder:validation:Minimum=-1
+	// logLevel specifies the verbosity level of the logging for a specific container.
+	// The "git-sync" and "otel-agent" containers default to 5, while all other containers default to 0.
+	// Increasing the value of logLevel increases the verbosity of the logs.
+	// Lower severity messages are logged at higher verbosity.
+	// Allowed values are from 0 to 10.
+	// +kubebuilder:validation:Minimum=0
```

Comment:
How do users know what number maps to what level, e.g. for `otel-agent`, 5 maps to info, 0 maps to fatal, etc.? Do we need to document them all or point to some standard/convention?

Reply:
The general convention here is that a higher log level will be more verbose and print lower severity messages, which is captured by the API description. I think it would be too much information to include the exact mappings in the API description here - we could add a user doc with more information.